doc/project/release_process.rst

```diff
@@ -85,24 +86,154 @@ At that point, the whole process starts over again.
+- All high and critical static analysis and security issues addressed
+- Footprint/Performance Tracking (no specific KPIs)
+- License audits pass
+- Documentation is complete and Release Notes are up-to-date.
+- Reproducible Builds (CI builds pass)
+
+Bug Severity and Priority
+=========================
+
+Priority of bugs is not enough to make release decisions: not all critical bugs
+are of highest priority and they do not need to be addressed in a release.
+Priority can also be set high on less critical bugs.
+
+Severity shall be used to classify bugs based on impact to users and community
+in general. For example:
+
+- A kernel bug impacting every user can be set as critical or major. If it needs
+  to be fixed in the next release, it will need to be marked as a blocker bug.
+- A Kconfig option or DTS in a board is impacting a small group of users, so it
+  could be minor or moderate but, a bug in hardware that is marked as a
+  `reference hardware` can be major or critical and in some cases a release
+  blocker.
+
+Bug priorities can be used by owners of bugs in an area or subsystem to manage
+their workflow and priorities.
+A low or moderate severity bug in a platform might be set to `High Priority` by the
+owner to manage execution within a subsystem or by a hardware vendor/maintainer.
+
+
+Severity
+--------
+The extent to which an issue or defect can affect the software and defines the
+impact that a given defect has on the system.
+
+The following severities shall be used:
+
++----------+------------------------------------------------------------------+
+| Severity | Description                                                      |
++==========+==================================================================+
+| Critical | Bug results in a crash of the system or one or more components   |
+|          | of the system. If the failed function is unusable and there is   |
+|          | no acceptable alternative method to achieve the advertised       |
+|          | desired outcome then the severity will be stated as critical.    |
++----------+------------------------------------------------------------------+
+| Major    | As with critical, but an acceptable alternative exists.          |
++----------+------------------------------------------------------------------+
+| Moderate | Causes the system to produce incorrect, incomplete, or           |
+|          | inconsistent results.                                            |
++----------+------------------------------------------------------------------+
+| Minor    | No crashes or damage to usability of the system and desired      |
+|          | results can be easily obtained by working around the defects.    |
++----------+------------------------------------------------------------------+
+| Cosmetic | Related to look and feel of the system.                          |
++----------+------------------------------------------------------------------+
```

Comment:
Feels a bit too granular to me, but guess we can go with it and see how it works out. Bet we'll only end up using few of these.

Reply:
I'll agree with @fabiobaltieri  here.  Not really sure there is a reason to have a difference between `Minor` and `Cosmetic` The rest of the classifications look correct to me.